docs/farming.md

```diff
@@ -61,10 +51,9 @@ Installing OpenCL GPU drivers or using farmer executable without `opencl` in fil
 # Replace `INSERT_YOUR_ID` with a nickname you choose
 # Copy all of the lines below, they are all part of the same command
 .\NODE_FILE_NAME.exe `
---chain gemini-1 `
+--chain gemini-2a `
 --execution wasm `
---blocks-pruning 1024 `
---state-pruning 1024 `
+--state-pruning archive `
```

Comment:
Is this a short-term change or a long-term change?

Reply:
This will last for the whole Gemini 2, long-term we'll not have it